app.js

```diff
@@ -625,14 +801,17 @@ class Player extends Entity {
 				const b = new Bullet({
 					parent: this.id,
 					angle: angle,
+					barrel: classes[classes[this.tank].barrels[0].bulletType].barrels,
+					tank: classes[this.tank].barrels[0].bulletType,
+					parent_tank: this.tank
 				});
 				b.x = this.x - 10;
 				b.y = this.y;
-				if (this.tank === "bomber" || this.tank === "grenadier") {
+				if (this.tank === "bomber" || this.tank === "grenadier" || this.tank === "rocketeer") {
```

Comment:
Why is Rocketeer set up as a tank that has exploding bullets?

Reply:
That was from another test.